src/content/weapons/ScytheOfVitur.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+import ScytheInventImage from "../../assets/images/weapons/scytheOfVitur.png"
+import { MeleeWeapon } from "../../sdk/Weapons/MeleeWeapon";
+import { Pathing } from "../../sdk/Pathing";
+
+export class ScytheOfVitur extends MeleeWeapon {
+  get attackRange() {
+      return 1;
+  }
+
+  get attackSpeed() {
+    return 5;
+  }
+
+  get inventoryImage() {
+    return ScytheInventImage;    
+  }
+
+  get bonuses() {
+    return {
+      attack: {
+        stab: 70,
+        slash: 110,
+        crush: 30,
+        magic: -6,
+        range: 0
+      },
+      defence: {
+        stab: -2,
+        slash: 8,
+        crush: 10,
+        magic: 0,
+        range: 0
+      },
+      other: {
+        meleeStrength: 75,
+        rangedStrength: 0,
+        magicDamage: 0,
+        prayer: 0
+      },
+      targetSpecific: {
+        undead: 0,
+        slayer: 0
+      }
+    }
+  }
+
+  // Scythe attacks in a 1x3 arc in front of the player.
+  // TODO: Refactor/change method so that it can actually hit multiple targets.
+  attack(region, from, to, bonuses = {}){
```

Comment:
I could be wrong, but I have the impression that the Scythe attacks the same target <n> times (without looking at location) for targets <n> by <n>, and it only does the  3x1 attack if it's multi-combat. I haven't done a ton of research into it though

Reply:
I say this because of bloat. You're almost always attacking from a corner, but the scythe still hits 3 times

Reply:
You're right, I missed over this bit: "Each hit can also hit large creatures (a target that takes up two or more game squares) up to three times in the same attack."